samples/simple_app/app.py

```diff
@@ -61,34 +51,55 @@ async def get_webhook_details(domain_name: DomainName) -> WebhookData:
         webhook_secret_key="webhook-secret-key",
     )
 
+async def example_dependency():
+    return "example"
+
 
-async def product_created(payload: Payload, saleor_domain: DomainName):
+async def product_created(payload: List[Webhook], saleor_domain=Depends(saleor_domain_header), example = Depends(example_dependency)):
```

Comment:
All webhook handler methods are now fully-fledged FastAPI endpoints, this allows dependency injection (which FastAPI related frameworks relly on)